openmmml/models/anipotential.py

```diff
@@ -63,27 +63,35 @@ def addForces(self,
                   atoms: Optional[Iterable[int]],
                   forceGroup: int,
                   filename: str = 'animodel.pt',
+                  useOptimizedTorchANI : Optional[bool] = True, 
                   **args):
         # Create the TorchANI model.
 
         import torchani
         import torch
         import openmmtorch
         if self.name == 'ani1ccx':
-            model = torchani.models.ANI1ccx()
+            model = torchani.models.ANI1ccx(periodic_table_index=True)
         elif self.name == 'ani2x':
-            model = torchani.models.ANI2x()
+            model = torchani.models.ANI2x(periodic_table_index=True)
         else:
             raise ValueError('Unsupported ANI model: '+self.name)
-
+        
         # Create the PyTorch model that will be invoked by OpenMM.
-
         includedAtoms = list(topology.atoms())
         if atoms is not None:
             includedAtoms = [includedAtoms[i] for i in atoms]
-        elements = [atom.element.symbol for atom in includedAtoms]
-        species = model.species_to_tensor(elements).unsqueeze(0)
-
+        atomic_numbers = [atom.element.atomic_number for atom in includedAtoms]
+        species = torch.tensor(atomic_numbers).unsqueeze(0)
+        
+        if useOptimizedTorchANI: # ask to optimize torchani
+            try:
+                # from https://github.com/openmm/NNPOps#example
+                from NNPOps import OptimizedTorchANI
+                model = OptimizedTorchANI(model, species)
```

Comment:
This should be in `ANIForce.__init__`, otherwise it won't be move to GPU. Check how @not-matt did that in #21.